frontend/src/pages/CreateRecording.tsx

```diff
@@ -3,130 +3,129 @@ import {MutableRefObject, Ref, useCallback, useEffect, useRef, useState} from "r
 import {EuiFieldText, EuiFlexGroup, EuiFlexItem} from '@elastic/eui';
 import React, { Component } from 'react'
 import EntitySearchBar from '../components/EntitySearchBar';
-import { Image, InteractionProgram } from '../components/Interfaces';
+import { InteractionProgram } from '../components/Interfaces';
 
 import prettyBytes from 'pretty-bytes';
 import { useNavigate } from 'react-router';
+import { AxiosRequestConfig } from 'axios';
+import { findAllImages, Image, useFindAllImages } from '../api';
+
+// const sendAPICall = useCallback(function () {
+//   setCommands(JSON.parse("[" + program + "]"))
+//   let recordingDetails = {
+//     volume: volume,
+//     commands: commands,
+//     name: name
+//   }
+//   console.log(recordingDetails)
+
+//   fetch('http://127.0.0.1:8080/panda', {
+//     method: 'POST',
+//     headers: {
+//       'Access-Control-Allow-Origin': '*',
+//       'Accept': 'application/json',
+//       'Content-Type': 'application/json'
+//     },
+//     body: JSON.stringify(recordingDetails)
+//   })
+//     .then(response => response.json())
+//     .then(response => displayResponse(response))
+// }, [name, volume, program])
+
+// const displayResponse = useCallback(function (response: any) {
+//   const term:any = terminal.current
+//   for (let i =0; i < response['response'].length; i++) {
+//     term.pushToStdout('panda@panda:~$ ' + commands[i] + '\n')
+//     term.pushToStdout(response['response'][i])
+//   }
+// }, [commands])
```

Comment:
Unless this chunk of comments has a purpose, it would be good to remove them to reduce clutter

Reply:
That is the code we were using for the beginning demo of running panda so I did not want to delete it quite yet but it will be removed in later cleanup work